packages/rn-tester/RNTesterPods.xcodeproj/project.pbxproj

```diff
@@ -847,6 +867,7 @@
 				ENABLE_BITCODE = NO;
 				ENABLE_STRICT_OBJC_MSGSEND = YES;
 				ENABLE_TESTABILITY = YES;
+				"EXCLUDED_ARCHS[sdk=iphonesimulator*]" = "arm64 ";
```

Comment:
Is `EXCLUDED_ARCHS` meant to be in quotes?

Reply:
This was added because we're now running `react_native_post_install` (https://github.com/facebook/react-native/blob/master/scripts/react_native_pods.rb#L127) like in the RN template project. I assume this is correct.

Reply:
I see. I was mainly curious if having: 

`"EXCLUDED_ARCHS[sdk=iphonesimulator*]" = "arm64 ";` 
vs
`EXCLUDED_ARCHS[sdk=iphonesimulator*] = arm64;`

makes a difference.

Reply:
not sure, maybe something there needs escaping? [ or *. Xcode does pickup the setting properly it looks like 
![image](https://user-images.githubusercontent.com/2677334/108786243-2c361d00-7541-11eb-94c6-652b010890ad.png)

Reply:
Alright cool, that's all that matters, merging 👍🏻.